packages/next/src/generators/infrastructure/utils/common.ts

```diff
@@ -176,20 +185,17 @@ export function addCommon(tree: Tree, options: NextGeneratorSchema) {
         options: {
             commands: [
                 {
-                    command: `helm upgrade --devel --create-namespace --install ${project.name} oci://${registryPaths.nonprod}/helm/${project.name} -n ${namespace} --atomic --wait --kube-context ${domainEnv.nonprod}-admin --set serviceAccount.annotations."azure\\.workload\\.identity/client-id"="{args.clientid}" --set serviceAccount.annotations."azure\\.workload\\.identity/tenant-id"="{args.tenantid}"`,
+                    command: `helm upgrade {args.helm-args} --create-namespace --install {args.values-files} ${project.name} {args.chart} -n ${namespace} --atomic --wait --kube-context {args.kube-context} --set serviceAccount.annotations."azure\\.workload\\.identity/client-id"="{args.clientid}" --set serviceAccount.annotations."azure\\.workload\\.identity/tenant-id"="{args.tenantid}"`,
                     forwardAllArgs: false,
                 },
             ],
-            cwd: `${project.root}/build/terraform`,
+            cwd: `${project.root}/deploy/helm/nonprod`,
+            args: `--helm-args=--devel --chart=oci://${registryPaths.nonprod}/helm/${project.name} --kube-context=${domainEnv.nonprod}-admin --values-files='--values values.yaml'`,
         },
         configurations: {
             prod: {
-                commands: [
```

Comment:
so prod doesn't need the commands array anymore?

Reply:
No, I removed the prod commands because the majority of it was the same as nonprod. I've utilised the `args` instead to pass in the values that change